test/CMakeLists.txt

```diff
@@ -276,5 +282,12 @@ endfunction()
 add_subdirectory(unit)
 add_subdirectory(integration)
 add_subdirectory(functional)
-add_subdirectory(package)
-add_subdirectory(example)
+
+# TODO: symlinks used! CK
+# package/pkg-config/src -> ../src/
+# package/FetchContent/src -> ../src/
+# package/find_package/src -> ../src/
+if (UNIX AND NOT PROJECT_IS_TOP_LEVEL)
```

Comment:
Please don't disable these. Even with symlinks they still run as shown in the other PR

Reply:
That does not work under windows, the symlinks and the installed export package is not found in that way you build this tests!

Reply:
Argh, I commented on the wrong PR, move this discussion to the main one